awx/ui_next/src/components/Lookup/Lookup.jsx

```diff
@@ -110,30 +130,31 @@ class Lookup extends React.Component {
 
     this.setState({ error: false });
     try {
-      const { data } = await getItems(queryParams);
+        const { data } = await getItems(queryParams);
       const { results, count } = data;
 
-      this.setState({
-        results,
-        count,
-      });
+        this.setState({
+          results,
+          count,
+        });
     } catch (err) {
       this.setState({ error: true });
     }
   }
 
   toggleSelected(row) {
-    const { name, onLookupSave, multiple } = this.props;
+    const { name, onLookupSave, multiple, onToggleItem, selectCategoryOptions } = this.props;
```

Comment:
I think the complexity in this function and be reduced once `Lookup` is refactored.